### PR TITLE
 Finish cleaning / fixing IDL documentation. [skip ci] 

### PR DESCRIPTION
**Pull request by phaazon**

### CODE REVIEW

File: core/idl/wallet/amount.djinni [hunk 1]
```diff
@@ -2,49 +2,58 @@
...
+    # The maximum number of decimals formatted decimals will have.
     maxNumberOfDecimals: i32;
 }
 
-#Class representing amount of transaction, output, inputs ...
+# Class representing amount of transaction, output, inputs…
 Amount = interface +c {
-    #Get amount as a BitInt
-    #@return BitInt
+    # Get amount as a BitInt.
```
Comment (KhalilBellakrid) [inline core/idl/wallet/amount.djinni]: Yay new type ! 🕺

File: core/idl/wallet/bitcoin/bitcoin_script.djinni [hunk 2]
```diff
@@ -12,8 +12,13 @@ BitcoinLikeScriptChunk = interface +c {
     hasNext(): bool;
 }
 
+# A general purpose script.
 BitcoinLikeScript = interface +c {
+    # Get the head of the script. Scripts are organized by chunks, so you get an iterator-like
+    # interface.
     head(): BitcoinLikeScriptChunk;
+    # Turn the script into astring representation.
```
Comment (KhalilBellakrid) [inline core/idl/wallet/bitcoin/bitcoin_script.djinni]: Yay ... and new representation  🕺

File: core/idl/wallet/common/wallet.djinni [hunk 3]
```diff
@@ -36,21 +36,32 @@ DerivationPath = interface +c {
...
+# It’s possible to either send funds or receive ones.
 OperationType = enum {
     send;
     receive;
 }
 
-# Enum of transaction level of trust.
+# Level of trust for transactions.
+#
+# A transaction is pending when it belongs to the last added block.
```
Comment (KhalilBellakrid) [inline core/idl/wallet/common/wallet.djinni]: Normally pending it's even not mined yet !

File: core/idl/wallet/amount.djinni (hunk 1 above)
Comment (phaazon) [reply core/idl/wallet/amount.djinni]: :D

**Pull request merged**